save-highscores.s

```diff
@@ -0,0 +1,60 @@
+.include "build/tetris.inc"
+.include "ips.inc"
+
+SRAM := $6000
+
+; patch to load from SRAM (and use a length instead of a terminator)
+
+.segment "initHighScoreTable_patch"
+        ips_segment     "initHighScoreTable_patch",$8095
+
+; initHighScoreTable_patch:
+initHighScoreTable:
+        lda SRAM,x
+        cpx #$50
+        beq continueColdBootInit
```

Comment:
I think normally I'd just stop here (since all the bytes will be the same).

Reply:
I think I still need the stuff after `beq continueColdBootInit` because I need to reference the label

Reply:
nevermind, I thought I'd changed it from bne to beq when I hadn't, it can come out